app/assets/javascripts/pages/data_portal/DataPortalCountryPage.js

```diff
@@ -57,18 +57,19 @@
       // NOTE: Do not set this property at instantiation time
       // The structure is:
       // [
-      //   { name: 'My indicator', options: ['Option 1', 'Option 2'] }
+      //   { id: 'My_indicator_id', name: 'My indicator', options: ['Option 1', 'Option 2'] }
       // ]
       // NOTE: There isn't any order guaranteed
       _indicatorsData: [],
       // List of active filters
       // Follow the same structure as _indicatorsData
       // NOTE: Do not set this property at instantiation time
-      _filters: []
+      _filters: [{}]
```

Comment:
Try without. If you absolutely need it for some reason, add a comment and explain why.